commit 9d5b626404b7cd616305d159aa898fa7a4ce8f69
Author: Dave Stevenson <dave.stevenson@raspberrypi.org>
Date:   Fri Sep 1 17:45:34 2017 +0100

    vcsm: Fix "void function return statements are not generally useful"
    
    Signed-off-by: Dave Stevenson <dave.stevenson@raspberrypi.org>

diff --git a/drivers/char/broadcom/vc_sm/vmcs_sm.c b/drivers/char/broadcom/vc_sm/vmcs_sm.c
index 4a1b22fe2ced..32333c6c23c1 100644
--- a/drivers/char/broadcom/vc_sm/vmcs_sm.c
+++ b/drivers/char/broadcom/vc_sm/vmcs_sm.c
@@ -984,8 +984,6 @@ static void vmcs_sm_host_walk_map_per_pid(int pid)
 	}
 
 	mutex_unlock(&(sm_state->map_lock));
-
-	return;
 }
 
 /*
@@ -1015,8 +1013,6 @@ static void vmcs_sm_host_walk_alloc(struct SM_PRIV_DATA_T *file_data)
 	}
 
 	mutex_unlock(&(sm_state->map_lock));
-
-	return;
 }
 
 /* Create support for private data tracking. */
